src/ray/core_worker/test/task_event_buffer_test.cc

```diff
@@ -58,7 +58,13 @@ class TaskEventBufferTest : public ::testing::Test {
 
   virtual void TearDown() {
     if (task_event_buffer_) task_event_buffer_->Stop();
-    std::filesystem::remove_all(log_dir_.c_str());
+    std::error_code ec;
+    std::filesystem::remove_all(log_dir_.c_str(), ec);
+    if (ec) {
+      std::cerr << "Error removing log_dir_ in TaskEventBufferTest teardown: "
```

Comment:
Does that mean on windows, we will have leaks?

Reply:
Yeah, we won't be deleting the `log_dir_` when it is in use by another process, but I thought that was fine because this only contains a single small file for export task events. Do we need to ensure the directory is always deleted?